rst/ketcher/release-notes/ketcher-2.7.0.md

```diff
@@ -0,0 +1,52 @@
+## What's Changed
+Release 2.7.0 is now available.
+Please be aware Ketcher 2.7.0 has been tested with Indigo version 1.8.3 ([standalone](https://www.npmjs.com/package/indigo-ketcher/v/1.9.0) and [remote](https://hub.docker.com/layers/epmlsop/indigo-service/1.9.0/images/sha256-c558047f1b359678e204fd9d32951d8c87b45e49c508b4b434a35ddb88c8e8d9?context=explore)).
+
+**Bug fixes**
+#2028 Hotkey: Atom symbols are not displayed under mouse cursor while mouse hovers over the canvas by @Nitvex in #2067
```

Comment:
please change ticket numbers to md format (link to ketcher github)
#2028 will not be recognized